draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -331,36 +373,49 @@ A catalog consumer SHOULD ignore custom properties it does not understand.
 As it is a regular DNS zone, a catalog zone can be transferred using DNS zone
 transfers among nameservers.
 
-Although they are regular DNS zones, catalog zones contain only information for
-the management of a set of authoritative nameservers.  For this reason,
-operators may want to limit the systems able to query these zones.  It may be
-inconvenient to serve some contents of catalog zones via DNS queries anyway due
-to the nature of their representation.  A separate method of querying entries
-inside the catalog zone may be made available by nameserver implementations
-(see (#implementationnotes)).
-
 Catalog updates should be automatic, i.e., when a nameserver that supports
 catalog zones completes a zone transfer for a catalog zone, it SHOULD apply
 changes to the catalog within the running nameserver automatically without any
 manual intervention.
 
-As with regular zones, primary and secondary nameservers for a catalog zone may
-be operated by different administrators.  The secondary nameservers may be
-configured as catalog consumer to synchronize catalog zones from the primary, but the primary's
-administrators may not have any administrative access to the secondaries.
-
 Nameservers MAY allow loading and transfer of broken zones with incorrect
-catalog zone syntax (as they are treated as regular zones), but catalog
-consumers MUST NOT process such broken zones as catalog zones.  For the purpose
-of catalog processing, the broken catalogs MUST be ignored.
+catalog zone syntax (as they are treated as regular zones).
+Catalog consumers MUST ignore broken catalog zones, however a clear error
+message stating the reason why the catalog zone is broken MUST be logged.
+
+When a previously correct catalog zone becomes a broken catalog zone, because
+of an update through an incremental transfer or otherwise, it loses its catalog
+meaning.
+No special processing occurs, no previously by this catalog configured member
+are removed or reconfigured in any way.
```

Comment:
Shall we use normative uppercase key words?

Reply:
Here RFC 2119 language makes sense to me yes.